docs/home-automation/home-assistant/presence-detection/detection-with-monitor-sh.md

```diff
@@ -0,0 +1,430 @@
+        payload: ''
+```
+
+Add the following configuration to automations.
+
+```yaml
+- alias: Set Person home
+  initial_state: 'on'
+  trigger:
+    - platform: state
+      entity_id: sensor.is_person1_home
+    - platform: homeassistant
+      event: start
+  condition:
+    - condition: state
+      entity_id: 'sensor.is_person1_home'
+      state: 'True'
+  action:
+    - service: script.person1_home
+
+- alias: Set Person1 away
+  initial_state: 'on'
+  trigger:
+    - platform: state
+      entity_id: sensor.is_person1_home
+    - platform: homeassistant
+      event: start
+  condition:
+    - condition: state
+      entity_id: 'sensor.is_person1_home'
+      state: 'False'
+  action:
+    - service: script.person1_away
+```
+This is where we link everything together. We also have the automations trigger when HA starts up. So we get some initial state.
+
+I personally also opted to have a BT scan triggered when ever HA restarts.
+
+I added this configuration to automations.
+```yaml
+alias: "HA Started"
+initial_state: 'on'
+trigger:
+  platform: homeassistant
+  event: start
+action:
+  - service: script.bt_rescan
+```
+
+The final step is the replace all the previous device_tracker references with the new manaully created ones listed in **known_devices.yaml**
+
+### Final Thoughts
+
+This is not yet an Ideal setup. I am still working on it. But if you look at the blog links below you might get some more ideas. I am still reading through them and improving things.
+
+
+As I stated at the start. This is mostly for me so that I have some documentation for myself.
+
+But I hope others might find it useful. Since I had to work through it and piece it together.
+
+Thanks to those who posted the work online. It was invaluable. The main part I found missing was the **known_devices.yaml**
+
+
+<hr/>
+
+### References:
+
+- [Bluetooth Presence Detection for Home Automation – The Level1 Way](https://forum.level1techs.com/t/bluetooth-presence-detection-for-home-automation-the-level1-way/148516)
+
+- [[monitor] Reliable, Multi-User, Distributed Bluetooth Occupancy/Presence Detection](https://community.home-assistant.io/t/monitor-reliable-multi-user-distributed-bluetooth-occupancy-presence-detection/68505)
+
+#### These are links to a single HA user. The config is far beyond what I need. But it's a great resource.
+- [Presence detection - are we nearly there yet?](https://blog.ceard.tech/2019/03/presence-detection-are-we-nearly-there.html)
+- [Presence detection, the final countdown?](https://blog.ceard.tech/2019/10/presence-detection-final-countdown.html)
+- [Presence detection updated](https://blog.ceard.tech/2018/09/a-while-back-i-covered-how-i-was-doing.html)
+- [GitHub Repo](https://github.com/DubhAd/Home-AssistantConfig)
+
+#### Some missing Magic.
+- [The missing magic to make MQTT State usable as a device_tracker](https://community.home-assistant.io/t/combining-multiple-device-trackers-into-one-using-mqtt/45324)
+
+### Software:
```

Comment:
Please remove `:` as this looks odd in the TOC on the right hand side

Reply:
Removed